torch-neuronx/training/fsdp_gpt2/run_clm_slurm.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+sudo rmmod neuron; sudo modprobe neuron
+sudo sysctl -w net.ipv4.ip_local_reserved_ports=44000,48620
+ulimit -c unlimited
+
+MASTER_ADDR=(`scontrol show hostnames $SLURM_JOB_NODELIST`)
+MASTER_PORT=2022
+NUM_NEURONCORES=32
+
+# if running inside slurm, handle here
+WORLD_SIZE_JOB=$SLURM_NTASKS
+RANK_NODE=$SLURM_NODEID
+JOB_ID_TAG=job-"$SLURM_JOB_ID"
+DISTRIBUTED_ARGS="--nproc_per_node $NUM_NEURONCORES --nnodes $WORLD_SIZE_JOB --node_rank $RANK_NODE --master_addr $MASTER_ADDR --master_port $MASTER_PORT"
+echo $DISTRIBUTED_ARGS
+
+
+export NEURON_RT_ROOT_COMM_ID=$MASTER_ADDR:46820
+export FI_EFA_USE_DEVICE_RDMA=1
+export FI_PROVIDER=efa
+
+# TODO : Is this needed ? enabled by default ? or disabled for debug only ?
+# os.environ['NEURON_RT_STOCHASTIC_ROUNDING_SEED'] = '0'
+export NEURON_RT_STOCHASTIC_ROUNDING_EN=1
+
+#Print Slurm Config
+date;hostname;
+#echo "WORLD_SIZE_JOB=$WORLD_SIZE_JOB,  RANK_NODE=$RANK_NODE,  MASTER_ADDR_JOB=$MASTER_ADDR_JOB, NODE_LIST=$NODE_LIST"
+
+
+export TRAINING_PRECISION=$1 #options FP32, BF16, MIXED
+
+if [[ "BF16" == $TRAINING_PRECISION ]]; then
+    echo "USING BF16 ONLY"
+    export XLA_USE_BF16=1
+    # need to clean up later, as 
+    # --distribution-strategy=FSDP
+    export NEURON_CC_FLAGS="--retry_failed_compilation --disable-internal-data-race-checker --internal-dram-page-size=2048 --enable-experimental-spmd --internal-allreduce-buffer-size=135 --model-type transformer --tensorizer-options=\'--enable-real-yk-ccop\' --internal-max-instruction-limit=10000000 --enable-experimental-O1  --internal-layers-per-module 4 --internal-ccop-bucketing --internal-ccop-bucketing-allgather-size-in-bytes 1 --internal-ccop-bucketing-reducescatter-size-in-bytes 62481600 --internal-ccop-bucketing-allreduce-size-in-bytes 62481600 --internal-build-with-users"
```

Comment:
The flags need to be cleaned up just like the `run_clm_neo.sh` file

Reply:
Thanks Jie! working on it..